docs/environment-variables.md

```diff
@@ -21,10 +21,9 @@ Env Var | Type | Default value | Required | Components | Description
  ADMIN_SESSION_EXPIRATION_TIME_IN_SECONDS | Int | 600 | Optional | AA | The session will expire after this many seconds of inactivity.
  APPINSIGHTS_COLLECT_DEPS | Boolean | true | Optional | AA | Set to false if you do not want App Insights to collect dependency information.
  APPINSIGHTS_COLLECT_EXCEPTIONS | Boolean | true | Optional | AA | Set to false if you do not want App Insights to collect exception information.
- APPINSIGHTS_INSTRUMENTATIONKEY | String | NULL | Optional | AA,PA | The Azure Application Insights account key, required for logging / monitoring.
  APPINSIGHTS_LIVE_METRICS | Boolean | true | Optional | AA | Set to false if you do not want App Insights to collection live metrics.
  APPINSIGHTS_WINSTON_LOGGER | Boolean | false | Optional | AA | Boolean flag.  Set to 1 to allow the winston logger to send to Application Insights, or 0 to disable. If enabled, you should also set `EXPRESS_LOGGING_WINSTON`to `1` and `APPINSIGHTS_INSTRUMENTATIONKEY` to the Azure-specified value.
- APPLICATIONINSIGHTS_CONNECTION_STRING | String | '' | Optional, but should be set for Prod | AA (Others to follow in v3 rollout) | Connection string for v3 of Application Insights node package.  This env var name is automatically supported by AI.
+ APPINSIGHTS_CONNECTION_STRING | String | '' | Optional, but should be set for Prod | AA, FT, FC, FP, PAPI | Connection string for v3 of Application Insights node package.  This env var name is automatically supported by AI.
```

Comment:
if its automatically picked up, should we be passing it in explicitly?

Reply:
Oh. That's an editing gap.  The 'automatically picked up' bit should have deleted when we did the name change, which is for consistency with other APPINSIGHTS env vars. 

I'll update it shortly.